code/gui/griddisplay.cpp

```diff
@@ -1,62 +1,97 @@
 #include <QDebug>
 #include "griddisplay.h"
+#include "../camera/cameradisplay.h"
 
 /* TODO (improvements):
- * Qt event handling is slow when handling 800 button slots
- * Change boolean to integer array, add drop down menu to select and set priorities
  * Allow user to select or deselect an area (using mouse events)
  * Dynamic implementation of columns/rows/button array
  * Set bounding rectangle of m_scene to m_img.size()
 */
 
+const QString buttonStyle =
+    "background-color: rgba(0, 0, 0, 0%);"  //clear
```

Comment:
continuation indents are an extra tab longer, and it might be better to use `#define` with these